plugins/callback/unixy.py

```diff
@@ -142,7 +142,7 @@ def v2_runner_on_ok(self, result, msg="ok", display_color=C.COLOR_OK):
             display_color = C.COLOR_CHANGED
             task_result = self._process_result_output(result, msg)
             self._display.display("  " + task_result, display_color)
-        elif self.display_ok_hosts:
+        elif self.get('display_ok_hosts'):
```

Comment:
@felixfontein shouldn't self.get_option be here as well?

Reply:
Hmm, yes, looks like a typo. Do you want to create a PR?

Reply:
@bobelev Thank you for finding this error! Yes, this is a typo I made.

I do wonder if my lack of testing did result in more errors.
I was assuming the change to be simple enough to let me do a drive-by contribution without looking into [collection_test_pr_locally](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_test_pr_locally.html) .

just created PR #5744 to fix the typo (but again  did not have the time to test the PR locally yet).

Reply:
#5744 fixes this.